api/v1alpha1/hostedcluster_types.go

```diff
@@ -697,6 +697,39 @@ type KubevirtPlatformCredentials struct {
 	InfraNamespace string `json:"infraNamespace"`
 }
 
+type KubevirtCSIDriverSpec struct {
+	// Disable prevents the passthrough of any infra cluster storage
+	// classes into the guest cluster.
+	//
+	// Defaults to false
+	// +optional
+	// +kubebuilder:validation:Optional
+	Disable bool `json:"Disable,omitempty"`
```

Comment:
Why is this needed?  Why not just delete all mappings?

Reply:
> Related question when mapping over the default storage class, what will it be named?

right now, the guest magically gets a storageclass called `kubevirt-csi-infra-default` by default

Reply:
>  think it would be best for this map (or something in status) to mirror what mappings actually exist. Also, instead of just a bool maybe an enum like None Default Explicit. Default could be Default and it would map the default storage class automatically. Explicit would be just what's in the map

I think an enum is a good idea here and makes this less confusing. I'll work with that a bit